sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -15,15 +15,21 @@ export const FastFrameTemplate = html<FastFrame>`
                 <fast-tab id="contrast-tab" title="Dark Mode">${ContrastIcon}</fast-tab>
                 <fast-tab id="palette-tab" title="Color">${PaletteIcon}</fast-tab>
                 <fast-tab id="style-tab" title="Styles">${SwatchesIcon}</fast-tab>
-                <fast-tab-panel id="contrast-tab-panel" class="${x =>
-                    x.expanded ? "tab-panel-expanded" : ""}">
+                <fast-tab-panel
+                    id="contrast-tab-panel"
+                    class="${x => (x.expanded ? "tab-panel-expanded" : "")}"
+                >
                     <div class="content">
-                        <h1><span class="content-heading-highlight">Fast Frame</span> Dark Mode</h4>
-                        <h2>Pre-built for both light and dark modes</h2>
+                        <p class="heading">
```

Comment:
Can you elaborate on how this is more semantic than using the correct heading tag here? This seems to bypass the issue of incorrect headings by removing any hierarchy altogether.

Reply:
I just thought we already have a section header and the headers in fast-frame could just be `p` elements but you are right, that it would not be semantically right, within the component itself. 
After further investigation, this looks similar to our sections, with a badge and a heading. The `h1` can be a badge, as an arbitrary element. The `h2` will be an `h3`.

`<fast-badge class="content-badge">
    Fast Frame
    <span class="content-badge-highlight">Dark Mode</span>
</fast-badge>
<h3 class="heading">Pre-built for both light and dark modes</h3>
`